### ISSUE TITLE
RangeError with WebAssembly version

### ISSUE THREAD

**Issue by s3ththompson** [mafintosh/siphash24#2]
SipHash complains of a RangeError in Node v8.5.0 with WebAssembly support. I'm calling SipHash through the [`buffer-graph`](https://github.com/yoshuawuyts/buffer-graph) library which is in turn part of [`bankai#new-new-new`](https://github.com/choojs/bankai/tree/new-new-new).

```
/usr/local/lib/node_modules/bankai/node_modules/siphash24/index.js:40
    mem.set(data, 24)
        ^

RangeError: Source is too large
    at Uint8Array.set (native)
    at siphash24 (/usr/local/lib/node_modules/bankai/node_modules/siphash24/index.js:40:9)
    at /usr/local/lib/node_modules/bankai/node_modules/buffer-graph/index.js:84:30
    at /usr/local/lib/node_modules/bankai/lib/graph-script.js:55:7
    at /usr/local/lib/node_modules/bankai/node_modules/browserify/index.js:779:13
    at ConcatStream.<anonymous> (/usr/local/lib/node_modules/bankai/node_modules/browserify/node_modules/concat-stream/index.js:36:43)
    at emitNone (events.js:110:20)
    at ConcatStream.emit (events.js:207:7)
    at finishMaybe (/usr/local/lib/node_modules/bankai/node_modules/browserify/node_modules/concat-stream/node_modules/readable-stream/lib/_stream_writable.js:475:14)
    at endWritable (/usr/local/lib/node_modules/bankai/node_modules/browserify/node_modules/concat-stream/node_modules/readable-stream/lib/_stream_writable.js:485:3)
```

**Comment by mafintosh** [comment]
Anyway you can share a reproducible test case?

**Comment by s3ththompson** [comment]
Let me see if I can isolate it... it's a closed source app that I'm running Bankai on.

**Comment by mafintosh** [comment]
Thanks! Feel free to dm me on freenode otherwise with a private test case (I can keep secrets 😁)

**Comment by s3ththompson** [comment]
added you to a private repo

**Comment by mafintosh** [comment]
should be fixed in 1.1.0
